android/build.gradle

```diff
@@ -26,7 +26,9 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 31
+    namespace 'com.survicate.flutter'
```

Comment:
We can remove `package="com.survicate.flutter"` from AndroidManifest

Reply:
or leave it in manifest and here wrap with `if (project.android.hasProperty("namespace"))`

Reply:
I removed package from Manifest